sdk/core/core-amqp/src/retry.ts

```diff
@@ -174,14 +204,30 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
         i,
         err
       );
+      let targetDelayInMs: number;
+      if (config.exponentialRetry) {
+        let incrementDelta = Math.pow(2, i) - 1;
+        const boundedRandDelta =
+          config.delayInSeconds * 0.8 +
+          Math.floor(Math.random() * (config.delayInSeconds * 1.2 - config.delayInSeconds * 0.8));
+        incrementDelta *= boundedRandDelta;
+
+        targetDelayInMs = Math.min(
+          config.minExponentialRetryDelayInMs + incrementDelta,
+          config.delayInSeconds
```

Comment:
We should be choosing between `config.minExponentialRetryDelayInMs + incrementDelta` and `maxExponentialRetryDelayInMs` not `config.delayInSeconds`.

Reply:
ah, yes - fixed